docs/intro.md

```diff
@@ -1,49 +1,121 @@
+
+    7. **Cutover**: Switch your clients to connect to the target Percona Server for MongoDB cluster.
+
+    For detailed information about sharded cluster replication, see [Sharding support in {{pcsm.full_name}}](sharding.md).
 
 ## Filtered replication
 
-You can replicate the whole dataset or only a specific subset of data, which is a filtered replication. You can use filtered replication for various use cases, such as:
+You can replicate the whole dataset or only a specific subset of data, which is a filtered replication. Filtered replication works for both replica sets and sharded clusters. You can use filtered replication for various use cases, such as:
 
-* Spin up a new development environment with a specific subset of data instead of the whole dataset. 
+* Spin up a new development environment with a specific subset of data instead of the whole dataset.
 * Optimize cloud storage costs for hybrid environments where your target MongoDB deployment runs in the cloud.
 
-Specify what namespaces - databases and collections - to include and/or exclude from the replication when you start it. 
+Specify what namespaces—databases and collections—to include and/or exclude from the replication when you start it. See [Start the filtered replication](install/usage.md#start-the-filtered-replication) for details.
+
+## Index handling
+
+{{pcsm.short}} manages indexes throughout the replication process to ensure data consistency and query performance on the target cluster.
+
+### Replication stage 
+
+During the replication stage, {{pcsm.short}} copies indexes from the source to the target cluster as follows:
+
+* **Unique indexes handling**: Unique indexes are copied as non-unique indexes during replication. This allows {{pcsm.short}} to handle potential duplicate data that may exist during the migration process.
+
+* **Hidden and TTL index handling**: {{pcsm.short}} copies hidden indexes as non-hidden. When copying TTL indexes, {{pcsm.short}} temporarily disables TTL expiration and saves the `expireAfterSeconds` property for later restoration. This way {{pcsm.short}} ensures documents won't expire while being copied.
+
+* **Index creation during sync**: If an index is created on the source cluster while the clusters are in sync, {{pcsm.short}} automatically creates the same index on the target cluster.
+
+* **Failed index creation**: If {{pcsm.short}}  cannot create an index during replication, it proceeds with replication and records the failure. The index creation will be retried during the finalization stage.
```

Comment:
We should add here the same for incomplete indexes. If the index build is in progress, PCSM will record that in the catalog and proceed with clone. In finalization PCSM will try to create them.